src/guide/ssr/getting-started.md

```diff
@@ -64,6 +64,8 @@ yarn add express
 ```
 
 ```js
+// server.js
```

Comment:
Since most of the people who want to migrate from CSR to SSR would be having a some kind of `main.js` entry file maybe it would make sense to show this migration path as a step-by-step guide how we extract our current code from `main.js` and split it into three different files: `app.js`, `entry-client.js` and `entry-server.js`. The current approach is also very good, but I think it's more focused on writing an SSR app from scracth. Maybe there should be a separate guide for migrating from CSR to SSR? What do you think?

Reply:
Agreed, it would be a great improvement! Created a follow-up: https://github.com/vuejs/docs-next/issues/970